suites/apis/etlTest.js

```diff
@@ -1,15 +1,26 @@
 Feature('ETL @requires-tube');
 
+const { expect } = require('chai');
+
 const { Bash } = require('../../utils/bash.js');
 const { checkPod } = require('../../utils/apiUtil.js');
 
 const bash = new Bash();
+let aliases = [];
 
 BeforeSuite(async ({ etl }) => {
-  etl.props.aliases.forEach(async (alias) => {
-    const index = etl.do.getIndexFromAlias(alias);
-    if (index !== 'error') {
-      etl.do.deleteIndices(index);
+  const etlMappingNames = bash.runCommand(`g3kubectl get cm etl-mapping -o jsonpath='{.data.etlMapping\\.yaml}' | yq '.mappings[].name' | xargs`).split(" ");
```

Comment:
is there any specific reason to do it before and after suite?

Reply:
`etl.ask.hasVersionIncreased(index, 0);` in `AfterSuite` requires that the index end with `_1`.

It's a bit of an edge case, but if indices exist before `etlTest.js` runs, those indices would be incremented to a version greater than `_1` (e.g. say `jenkins-dcp.planx-pla.net_subject_1` isn't deleted because the pod running the integration tests crashed after the ETL runs but before index cleanup in `AfterSuite`, then on the next `jenkins-dcp` run, `jenkins-dcp.planx-pla.net_subject_2` would be created and `etl.ask.hasVersionIncreased(index, 0);` would fail).

Reply:
I thought that function's purpose is to check that ETL ran the second time fine, not sure why it is in AfterSuite though, will check the code

Reply:
I can move `etl.ask.hasVersionIncreased(index, 0);` to the last `Scenario`. That way, we can use `cleanUpIndices` in `AfterSuite`.